test/cases/aws.sh

```diff
@@ -216,7 +216,7 @@ fi
 
 greenprint "Pulling cloud-image-val container"
 
-CONTAINER_CLOUD_IMAGE_VAL="quay.io/cloudexperience/cloud-image-val-test:latest"
```

Comment:
Alternatively, you could be pushing the latest container built from main under the `latest` tag to the registry, no? I don't have anything against this change, but IMO it makes sense for the `latest` tag to point to the newest build of the container, which perhaps should be the one built from `main`...

Reply:
This is confusing I know :S. The reason we push to main and not latest is to avoid code duplication in CIV CI. We are not going to use the :latest tag anymore.

Reply:
I just realized it won't be that hard to fix this. I just have to keep an eye out after a few systems that depend on latest. Thanks!

Reply:
OK, fair enough.